### DISCUSSION TITLE
I want to buy the plan

### DISCUSSION THREAD

**Discussion by GearStarter** [community/community#22970] [New to GitHub]
<p>Hi, admins GitHub!</p>
<p>I need repo with <strong>100 GB</strong> for my project.</p>
<p>What plan should I buy?</p>

**Comment by ovflowd** [comment, marked as answer]
<p>Hey <a class="mention" href="/gearstarter">@gearstarter</a> welcome to the Community Forum :smile:</p>
<p>GitHub has a 100GB a storage limit for your repository. If you have  <strong>large</strong> files (more than 100MB each), then you need to configure your Git (locally) and prepare the repository remotely to support Git LFS (Large File Storage)</p>
<p>You can check the guide here <a href="https://help.github.com/en/github/managing-large-files/configuring-git-large-file-storage" rel="nofollow noopener">https://help.github.com/en/github/managing-large-files/configuring-git-large-file-storage</a></p>
<p>Also check the Disk Quota here <a href="https://help.github.com/en/github/managing-large-files/what-is-my-disk-quota" rel="nofollow noopener">https://help.github.com/en/github/managing-large-files/what-is-my-disk-quota</a></p>
<p>I do not believe you ca exceed 100GB.</p>

**Comment by GearStarter** [comment]
<p>Thanks!</p>
<p>I’m game developer. I can. :smile:</p>

**Comment by ovflowd** [comment]
<p>Oh yes, games can quickly get bigger than 100GB, But I meant that I think that GitHub doesn’t allow in anyway repositories with more than 100GB.</p>
